src/index.html

```diff
@@ -6,9 +6,54 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet"
+    href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="logo">
+        <a class="logo" href="#">
+          <img src="images/logo.png" alt="moyo">
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a href="#" class="nav-link is-active">Apple</a>
+          </li>
+          
+          <li class="nav-item">
+            <a href="#" class="nav-link">Samsung</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">Smartphones</a>
+          </li>
+          <li class="nav-item">
+            <a
+            href="#"
+             data-qa="hover"
+              class="nav-link"
+              >
+              Laptops & Computers
+            </a>
```

Comment:
code styling from a checklist
```suggestion
            <a
              href="#"
              data-qa="hover"
              class="nav-link"
            >
              Laptops & Computers
            </a>
```

Reply:
The same